app/cmd/controller.go

```diff
@@ -85,6 +90,7 @@ func startController(c *cli.Context) error {
 	isUpgrade := c.Bool("upgrade")
 	disableRevCounter := c.Bool("disableRevCounter")
 	salvageRequested := c.Bool("salvageRequested")
+	unmapMarkSnapChainRemoved := c.Bool("unmap-mark-snap-chain-removed")
```

Comment:
Does longhorn-engine need to update the cli version due to the change?

Reply:
IIRC, you already bump up the version for v1.4.0 in other PRs, then I don't need to handle that again.

Reply:
Got it. Thank you.